client/client.go

```diff
@@ -222,6 +222,10 @@ func WithCredentials(serverName, ca, cert, key string) ClientOpt {
 }
 
 func loadCredentials(opts *withCredentials) (grpc.DialOption, error) {
+	if opts.CACert == "" {
+		return grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})), nil
```

Comment:
can we add an earthly-specific comment, which will make future upstream merging into earthly-main easier:

```suggestion
		return grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})), nil // earthly-specific: make use of default TLS settings (but still forces a TLS connection)
```